src/app/shared/devices/device-list/device-list.component.ts

```diff
@@ -17,8 +18,9 @@ const headersForDisplay = [
 })
 export class DeviceListComponent {
     @Input() deviceList: Array<Device>;
+    @Input() editOnItemClick: boolean = true;
     @Input() showDeleteButton: boolean = true;
-    @Input() showStatisticLink: boolean = true;
+    @Input() statisticLink: string|boolean = false;
```

Comment:
I don't know if I can assign to the property true or false value. It seems that 'false' value uses only for initial state. And 'true' - never.